content/en/armory-enterprise/quick-spin-instructions/_index.md

```diff
@@ -4,28 +4,37 @@ linkTitle: "quick-spin-instructions"
 no_list: true
 weight: 3
 description: >
-  Get an evaluation or testing instance of Spinnaker up and running.
+  Thank You for signing up to Quick Spin. Use these instructions to evaluate Armory's Continuous Deployment Self-Hosted solution, or get help from our team [Contact Us](https://www.armory.io/contact-us/).
 exclude_search: true
 toc_hide: true
 ---
-{{< include "experimental-feature.html" >}}
 
-## What is Quick Spin?
-Quick Spin is a containerized Armory CD instance that you install with a single shell command. You can use this minimal instance to evaluate the Armory CD Self-Hosted product.
+> The Quick Spin container is not intended for production use.
 
-Once you have Quick Spin running, you can use it to evaluate Armory CD Self Hosted and to learn how to use Spinnaker. You can also use Quick Spin to test out Spinnaker features and to learn how to configure Spinnaker. 
+{{% heading "prereq" %}}
 
-Included with the Quick Spin container is a sample application and three sample pipelines that you can use to test out Spinnaker. After installing Quick Spin, you can use the sample application and pipelines to test out Spinnaker features.
+To run the Quick `quick-spin` container you need a host machine that can support a running instance with 
+  - 4 vCPU
+  - 8GB Memory
+  - 2GB Disk space
+  - The host machine must have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed. Docker Compose is installed with Docker Desktop.
 
+## Installing Quick Spin
 
-> The Quick Spin container is not intended for production use.
+Open a terminal to the preferred installation directory and run the following docker command:
 
-## Installing Quick Spin
+```shell
+ docker compose pull quickspin armory/quickspin:latest
+```
+{{< alert color="success" >}}Quick Spin is running on `localhost:9000`. 
+The provided instance deploys a basic NGINX instance and sets up a minimal deployment configuration. Now you can start using Spinnaker to deploy your own applications.{{< /alert >}}
 
-- To install Quick Spin use the steps provided in {{< linkWithTitle "install-quick-spin" >}}. 
-- See the {{< linkWithTitle "quickstart-quick-spin" >}} to get started using Quick Spin.
+## What can I do with Quick Spin?
+Quick Spin is a containerized Armory CD instance that you install with a single shell command. You can use this minimal instance to evaluate the Armory CD Self-Hosted product.
+
+Once you have Quick Spin running, you can use it to evaluate Armory CD Self Hosted and to learn how to use Spinnaker. You can also use Quick Spin to test out Spinnaker features and to learn how to configure Spinnaker. 
```

Comment:
L35 is saying almost the same thing as L33, so how about omitting L 35 and adding a bit to L33?
Something like

```
Quick Spin is a containerized Armory CD instance that you install with a single shell command. You can use this minimal instance to evaluate the Armory CD Self-Hosted product in functional areas such as configuration and deploying an application.
```